webui/react/src/pages/WorkspaceDetails/ProjectCard.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import Avatar from 'components/Avatar';
+import Icon from 'components/Icon';
+import Link from 'components/Link';
+import TimeAgo from 'components/TimeAgo';
+import { paths } from 'routes/utils';
+import { DetailedUser, Project } from 'types';
+
+import ProjectActionDropdown from './ProjectActionDropdown';
+import css from './ProjectCard.module.scss';
+
+interface Props {
+  curUser?: DetailedUser;
+  project: Project;
+}
+
+const ProjectCard: React.FC<Props> = ({ project, curUser }: Props) => {
+
+  return (
+    <ProjectActionDropdown curUser={curUser} project={project}>
+      <div className={css.base}>
+        <h6 className={css.name}>
+          <Link inherit path={paths.projectDetails(project.id)}>
+            {project.name}
+          </Link>
+        </h6>
+        {!project.immutable && (
+          <ProjectActionDropdown className={css.action} curUser={curUser} project={project} />
+        )}
+        <p className={css.description}>{project.description}</p>
+        <div className={css.experiments}>
+          <Icon name="experiment" size="small" />
+          <span>{project.numExperiments}</span>
```

Comment:
<img width="304" alt="Screen Shot 2022-04-15 at 3 29 11 PM" src="https://user-images.githubusercontent.com/643918/163649475-8aa491ab-bd90-4a88-8fd7-43638ebc540a.png">
nit picks: Could this be `.toLocaleString()` (for >1,000 experiments) and some margin or padding

Reply:
<img width="383" alt="Screen Shot 2022-04-18 at 10 29 56 AM" src="https://user-images.githubusercontent.com/643918/163848049-d7b0edc7-bae7-484e-8752-eb63c6ef5058.png">

I still have an issue with no whitespace between the number and the time, it looks like the position is set by the grid and any more digits would make the count and time overlap.

Reply:
Alright, I fixed this for real this time.